app/src/components/blocks/feedbackForm/FeedbackForm.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+    FORM_FIELDS,
+    FormData,
+    SATISFACTION_CHOICES,
+    SUBMISSION_STAGE,
+} from '../../../types/pages/feedbackPage/types';
+import React, { Dispatch, useState } from 'react';
+import { SubmitHandler, useForm, UseFormRegisterReturn } from 'react-hook-form';
+import sendEmail from '../../../helpers/requests/sendEmail';
+import { Button, Fieldset, Input, Radios, Textarea } from 'nhsuk-react-components';
+import SpinnerButton from '../../generic/spinnerButton/SpinnerButton';
+
+export type Props = {
+    readonly stage: SUBMISSION_STAGE;
+    readonly setStage: Dispatch<SUBMISSION_STAGE>;
```

Comment:
Props in React are immutable by default, this is an anti-pattern and not necessary

Reply:
This is to suppress a so-called "Code Smells" that SonarCloud complains about. I am happy to remove the readonly if its consider an anti-pattern.
 https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&id=prm-repository_repository&open=AYwfjaMMhuX2Ulp5DlkP

Reply:
I wonder if sonar cloud complained because the setStage prop doesn't have the 'SetStateAction' part we have used elsewhere in the code eg setStage: Dispatch<SetStateAction<LG_RECORD_STAGE>>;
could try adding that and removing readonly to see if it resolves the 'code smell'